Procfile

```diff
@@ -1,3 +1,3 @@
-web: app/app.py --config=app/config.py
-db: mongod --dbpath data/db
+web: python app/app.py --config=app/config.py
+db: mongod --dbpath c:\mongodb\data
```

Comment:
Yeah I agree.  I did make this in a separate branch, so if we wanted both in source control we could do that.  If you'd rather it just not be checked in that's fine as well with me, not like it's hard to keep track of/change, plus as you say it would be documented in the README.